controllers/etcd/config.go

```diff
@@ -15,9 +15,11 @@
 package etcd
 
 import (
-	"flag"
-
 	"github.com/gardener/etcd-druid/controllers/utils"
+	"github.com/gardener/etcd-druid/pkg/features"
+
+	flag "github.com/spf13/pflag"
```

Comment:
Can you clarify why the default `flag` package is replaced with `pflag` package for this as well as other controller configs?

Reply:
`pflag` was needed in order to add `--feature-gates` to the flagset as can be seen from the implementation of [`featuregate.AddFlag(fs)`](https://github.com/kubernetes/component-base/blob/v0.27.4/featuregate/feature_gate.go#L321)
Since `pflag` was being used, it made sense to use the same package throughout the repo instead of having multiple packages

Reply:
Got it, thanks!